src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/BrevPeriodeContext.kt

```diff
@@ -366,3 +383,17 @@ class BrevPeriodeContext(
         }
     }
 }
+
+private fun Begrunnelse.hentRelevanteEndringsperioderForBegrunnelse(
+    endretUtbetalingAndeler: List<EndretUtbetalingAndel>,
+    vedtaksperiode: UtvidetVedtaksperiodeMedBegrunnelser
+): List<EndretUtbetalingAndel> = when (this.begrunnelseType) {
+    BegrunnelseType.ETTER_ENDRET_UTBETALING -> {
+        endretUtbetalingAndeler.filter {
+            it.periode.tom.sisteDagIInneværendeMåned()
+                ?.erDagenFør(vedtaksperiode.fom?.førsteDagIInneværendeMåned()) == true
+        }
+    }
```

Comment:
Skal vi ikke se på endringsårsak her?

Reply:
Denne er shamelessly stjålet fra ba-sak, og de sjekket ikke for det der 🤔

Reply:
La til sjekk her 👍